packages/svelte-vscode/package.json

```diff
@@ -51,6 +51,11 @@
                     "title": "Language Server Runtime",
                     "description": "Path to the node executable to use to spawn the language server"
                 },
+                "svelte.language-server.ls-path": {
+                    "type": "string",
+                    "title": "Language Server Path",
+                    "description": "Path to the langauge server file"
```

Comment:
Could you add more details here? Like "either relative to the workspace root, or absolute. Use it when ...". Could you also add the same to the README (there is a settings section)?